ci/scripts/backfill-test.sh

```diff
@@ -32,4 +32,4 @@ download_and_prepare_rw "$profile" source
 
 ################ TESTS
 
-profile=$profile ./ci/scripts/run-backfill-tests.sh
+BUILDKITE=${BUILDKITE:-} profile=$profile ./ci/scripts/run-backfill-tests.sh
```

Comment:
May I ask what this change is for?

Reply:
It's to parameterize the script, so it runs differently locally and in ci.

In ci: cluster configuration does not include monitoring stack.
Locally: cluster configuration includes monitoring.